packages/patternfly-4/react-core/src/components/Pagination/OptionsMenu.js

```diff
@@ -0,0 +1,99 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import styles from '@patternfly/patternfly/components/OptionsMenu/options-menu.css';
+import { css } from '@patternfly/react-styles';
+import { Dropdown, DropdownItem, DropdownDirection } from '../Dropdown';
+import { CheckIcon } from '@patternfly/react-icons';
+import OptionsToggle from './OptionsToggle';
+
+
+const propTypes = {
+  className: PropTypes.string,
+  widgetId: PropTypes.string,
+  dropDirection: PropTypes.oneOf(Object.values(DropdownDirection)),
+  perPageOptions: PropTypes.arrayOf(PropTypes.shape({
+    title: PropTypes.node,
+    value: PropTypes.number
+  })),
+  titles: PropTypes.shape({
+    items: PropTypes.string
+  }),
+  itemCount: PropTypes.number,
+  firstIndex: PropTypes.number,
+  lastIndex: PropTypes.number,
+  perPage: PropTypes.number,
+  onPerPageSelect: PropTypes.func
+};
+const defaultProps = {
+  dropDirection: DropdownDirection.down,
+  titles: {}
+};
+
+class OptionsMenu extends Component {
+  state = {
+    isOpen: false
+  }
+
+  onToggle = isOpen => {
+    this.setState({
+      isOpen
+    });
+  };
+
+  onSelect = event => {
+    this.setState({
+      isOpen: !this.state.isOpen
+    });
+  };
+
+  renderItems = () => {
+    const { perPageOptions, onPerPageSelect, perPage } = this.props;
+    return perPageOptions.map(({ value, title }) =>(
+      <DropdownItem key={value}
+        component="button"
+        data-action={`per-page-${value}`}
+        className={css(styles.optionsMenuMenuItem, perPage === value && 'pf-m-selected')}
```

Comment:
modifier not available from the css object?

Reply:
Sadly options menu does not have `selected` modifier.

Reply:
@mcoker can we add this modifier to the Options Menu?  Seems like we would want that.  We can open up another react issue to make the update if core end up adding that.